system_strings/sysmes_text.en

```diff
@@ -4258,219 +4331,354 @@ I can't betray her trust.
 -- 炎が走り抜ける。^それはノエルを焼くばかりか、^彼女の身体を引き込んでいく。
-炎が走り抜ける。^それはノエルを焼くばかりか、^彼女の身体を引き込んでいく。
+// @flowchart_descr
+Fire floods the hallway.
+ Not content to simply burn Noel, he pulls her to him.
 }
 [sha:d3f58df86ea45663206f2650f25c325d16a21368]{
 -- 忘れじの声がする。^見過ごす事はできなかった。^頭で考えるより先に体が動いた。
-忘れじの声がする。^見過ごす事はできなかった。^頭で考えるより先に体が動いた。
+// @flowchart_descr
+I hear a voice I'll never forget.
+ I can't let this to happen.
+ My body starts moving before my mind can catch up.
 }
 [sha:f3235a0598407005b73019ac4cf24dbe7fc370ba]{
 -- ヴローヴはこちらを気にしない。^奇襲は成功するだろう。^このように。
-ヴローヴはこちらを気にしない。^奇襲は成功するだろう。^このように。
+// @flowchart_descr
+Vlov doesn't register me as a threat.
+ I can probably take him by surprise.
+ Just like this――
 }
 [sha:1ebdcf30edfa6f00fd4b5ff535b1ebbbe75bc074]{
 -- 忘れじの声がする。^しかし、ここからでは^あまりにも遠すぎた。
-忘れじの声がする。^しかし、ここからでは^あまりにも遠すぎた。
+// @flowchart_descr
+I hear a voice I'll never forget.
+ But, he's too far away for me to do anything.
 }
 [sha:4166b72f04a1c4be0fae42fd9c431e57300ad3e3]{
 -- このまま押し切って終わらせる。^あと一歩踏み込むだけでいい。^だが、過信の結果は常に非情だ。
-このまま押し切って終わらせる。^あと一歩踏み込むだけでいい。^だが、過信の結果は常に非情だ。
+// @flowchart_descr
+If I keep on going like this I can push through.
+ I just need to take one more step.
+ But the price of overconfidence is steep.
 }
 [sha:8a58d7d6c6b58d4061cb04721f6509bce9567e1a]{
 -- ヤツの能力はホテルで見ている。^ここは一旦距離を取らなければ。^だが、蛮勇の代償は……。
-ヤツの能力はホテルで見ている。^ここは一旦距離を取らなければ。^だが、蛮勇の代償は……。
+// @flowchart_descr
+I got a glimpse of his abilities back at the hotel.
+ I've got to keep my distance.
+ But the price for my recklessness is...
 }
 [sha:757ce7ba8cd98b3e0d46a9b64b66e38441b77772]{
 -- シエル先輩は失敗した。^吹き荒れる白い呪い。^広間は完全に凍り付く――
-シエル先輩は失敗した。^吹き荒れる白い呪い。^広間は完全に凍り付く――
+// Not really a 'failure' so much as the ole vlov cold/hot mistake
+// @flowchart_descr
+Ciel-senpai has made a mistake.
+ A tempestuous white curse.
+ The underground chamber is frozen solid.
 }
 [sha:87d8e80cd96869930f50d8179a059ff7d6a0e2e6]{
 -- 先輩を助けるに決まってる。^無我夢中でその背中を追う。^ここで逃げる事はできない……！
-先輩を助けるに決まってる。^無我夢中でその背中を追う。^ここで逃げる事はできない……！
+// @flowchart_descr
+Obviously I'm going to go save Senpai.
+ I frantically sprint toward her back.
+ I can't escape from this!
 }
 [sha:83b47873fe184b1d0d6b6e6cb101c10d3d7ca832]{
 -- ノエル先生を振り返る。^決めていた筈だ。^自分に出来る事をしなければ！
-ノエル先生を振り返る。^決めていた筈だ。^自分に出来る事をしなければ！
+// @flowchart_descr
+I look back at Noel.
+ I've already made my decision.
+ I have to do what I can!
 }
 [sha:27c3ae890805c69dc8367ebe69c6187f6092d29b]{
 -- ドームの崩落に巻き込まれた。^目覚めた場所は暗く、狭く、^そして柔らかい……柔らかい？
-ドームの崩落に巻き込まれた。^目覚めた場所は暗く、狭く、^そして柔らかい……柔らかい？
+// @flowchart_descr
+The underground chamber collapses around me.
+ When I wake up, the world around me is cramped, dark, and soft... soft?!
 }
 [sha:3dd7c368545243f353fd970a61dcb5f71ee8e470]{
 -- ようやく地上に戻る事ができた。^沈みつつある日の光さえ眩しい。^そして、先輩の視線は冷たい。
-ようやく地上に戻る事ができた。^沈みつつある日の光さえ眩しい。^そして、先輩の視線は冷たい。
+// 眩しい tweaked a bit to better mirror 冷たい
+// @flowchart_descr
+We finally make it back to the surface.
+ Even the faint rays of the setting sun feel warm.
+ However, Senpai's gaze is frosty.
```

Comment:
frosty isn't really what you'd use to describe someone's gaze, I think.
>In contrast, Senpai's gaze is chilling.

Reply:
It's a collocation, though surprisingly not as common as I thought.